contracts/libraries/GeneralAsset.sol

```diff
@@ -0,0 +1,37 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import { IERC20 } from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import { SafeERC20 } from "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+import { Address } from "@openzeppelin/contracts/utils/Address.sol";
+
+import { Constant } from "./Constant.sol";
+
+library GeneralAsset {
```

Comment:
nit: I personally would prefer to call it just `Asset`.   `General` doesn't bring additional meanings to me for this library. 😆

And it would be better to remove the function prefix also, and try to figure out more appropriate names for them. 

For instance:

* `generalBalanceOf` -> `getBalance`
* `generalTransfer` -> `transferTo`

Reply:
I'm ok with the naming. It can be adjusted later as well.
Fixed in 59d2abfdee82605b53e91593a18c9a77a217afd2